packages/paste-website/src/pages/tokens/theme-package.mdx

```diff
@@ -0,0 +1,146 @@
+| ------------------ | -------- | ----------------------------------------------------------------------------------------- | ----------------- |
+| theme?             | enum     | 'default',  'sendgrid'                                                                    | 'default'         |
+| customBreakpoints? | string[] | An optional array of string values for custom screen sizes in the usual CSS width formats | theme.breakpoints |
+
+## Using tokens in custom components
+
+Sometimes you will need to create something custom that is not available in Paste but you need that component to still look like it's from Paste. The use of Paste Tokens in your styles is the way that you can access global design properties used in all Paste Components.
+
+### Emotion Styled Components (recommended)
+
+By using the `Theme.Provider`, when you create a custom component using Emotion CSS, the theme object is available on `props` via context. You can access those via the [Styled-System `themeGet` utility](https://styled-system.com/api#themeget) or props object directly.
+
+```js
+import styled from '@emotion/styled';
+import { themeGet } from 'styled-system';
+
+const custom = styled.div`
+  background-color: ${themeGet('backgrounds.primary')};
+  padding: ${props => props.theme.spacing.spacing20};
+`
+```
+
+### Alternatives when not using Emotion
+
+If you are not using Emotion styled components or want to access the values of tokens in your React components outside of styling, there are three ways this can be achieved.
+
+#### Paste ThemeConsumer
+
+Paste Theme provides a React Context Consumer for the theme called `Theme.Consumer`. The `Theme.Consumer` takes a function as a child, which provides the theme object as an argument.
+
+```js
+import {Theme} from '@twilio-paste/theme';
+
+<Theme.Consumer>
+  {({theme}) => {
+    return <p>What is the default text color {theme.textColors.colorText}</p>;
+  }}
+</Theme.Consumer>
+```
+
+#### Paste useTheme Hook
+
+Paste Theme provides a React Hook called `useTheme` which returns the theme object from the React Context via the Theme Provider.
+
+```js
+import React from 'react';
+import { useTheme } from '@twilio-paste/theme';
+
+const HookExampleComponent = (): React.ReactElement => {
+  const theme = useTheme();
+  return <p>What is the default text color {theme.textColors.colorText}</p>;
+};
+```
+
+#### Higher Order Component
+
+Paste also provides a HOC Component, which is a wrapper around the Emotion [HOC Component](https://emotion.sh/docs/emotion-theming#withthemecomponent-reactcomponenttype-reactcomponenttype) to be able to access the theme object.
+
+```js
+import React from 'react';
+import { withTheme } from '@twilio-paste/theme';
+
+const ExampleComponent = ({theme}) => (
+  <p>What is the default text color {theme.textColors.colorText}</p>
+);
+const ExampleComponentwithTheme = withTheme(ExampleComponent);
+```
+
+***
+
+<Box marginTop={30}>
```

Comment:
Does this use the token or is it setting 30px? 😕

Reply:
Error on my part when I added the changelogs. It should be `space90`.